tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -87,3 +87,10 @@ Feature: An administrator wants to import a user using the commandline
     When a user is imported from path "simpleExport/unknown" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "user.json not found"
+
+  @issue-210
+  Scenario: importing with empty trash bin
+    When a user is imported from path "emptyTrashbinExport/usertrash" using the occ command
```

Comment:
Can't we use this user here? Is it necessary to create a new one and add all the files?
```suggestion
    When a user is imported from path "simpleExport/testUser" using the occ command
```